buildkitd/buildkitd.go

```diff
@@ -30,6 +31,8 @@ import (
 	"github.com/earthly/earthly/util/semverutil"
 )
 
+const minRecommendedCacheSize = 10 << 30 // 10 GiB
```

Comment:
`10.74 GB` looks a bit strange. Shall we use simply `10,000,000,000`?

Reply:
That's fine by me. Some of the other code was using GiB so I assumed that was the preference.